packages/Webkul/Shop/src/Resources/assets/js/components/image-slider.vue

```diff
@@ -67,7 +67,7 @@ export default {
                 self.images.push({
                     'path': self.public_path + '/storage/' + slider.path,
                     'title' : slider.title,
-                    'slider_path': slider.slider_path ? self.public_path+'/'+slider.slider_path : null
```

Comment:
@jaseelbavu kindly remove these changes as this is related to another issue.

Reply:
Done :+1: @vaishaliwebkul